CHANGELOG.md

```diff
@@ -2,7 +2,13 @@
 
 ## v3.1.2
 
-### `@liveblocks/react-ui` and `@liveblocks/emails`
+### `@liveblocks/react-ui`
```

Comment:
[v3.1.2](https://github.com/liveblocks/liveblocks/releases/tag/v3.1.2) is already out, v3.2 might be far away so feel free to start v3.1.3.